utils/generateUsers.ts

```diff
@@ -0,0 +1,97 @@
+import faker from 'faker';
+import { v4 as uuidv4 } from 'uuid';
+
+type GeneralProperties = {
+  id: string;
+  login: string;
+  password: string;
+  age: number;
+}
+
+export type User = GeneralProperties & {
+  isDeleted: boolean;
+}
+
+export class UserManager {
```

Comment:
it's more like `model`, i suggest to put in separate file in `models/UserModel`

Reply:
Fixed